README.md

```diff
@@ -1,133 +1,414 @@
-# SQLite Foreign Data Wrapper for PostgreSQL
-This PostgreSQL extension is a Foreign Data Wrapper for [SQLite][1].
+SQLite Foreign Data Wrapper for PostgreSQL
+==========================================
 
-The current version can work with PostgreSQL 11, 12, 13, 14 and 15.
+This is a foreign data wrapper (FDW) to connect [PostgreSQL](https://www.postgresql.org/) 11+
```

Comment:
When new PostgreSQL is released, it might not support it automatically.   
So please describe the upper limit of the version.

Reply:
Corrected. Is there any tests against SQLite versions? I think FDW supports SQLite 3.

Reply:
Thank you for fixing.

There is no test confirming SQLite version. Anyway, we used SQLite 3.38.5.

Reply:
Thanks, added in https://github.com/mkgrgis/sqlite_fdw/commit/5942c1ca9f21251d60cbb531a3960dc5bb9a643c